bitstyles/bitstyles/settings/_grid.scss

```diff
@@ -9,11 +9,7 @@ $bitstyles-widths-columns-responsive: (12) !default;
 // breakpoint.
 // If this is not defined, no responsive breakpoints will be generated.
 $bitstyles-grid-breakpoints: (
-  'small',
-  'medium',
-  'medium-and-up',
-  'small-and-medium',
-  'large'
+  'medium'
```

Comment:
`medium-and-up` would be more mobile-first (as a default, lets make it as correct as possible!)